### PR TITLE
Support psql older than 8.4

### LINKED ISSUE #270: psql 8.3 and before doesn't accept -w
This makes check_pga currently not compatible with such versions of psql.

### PR DESCRIPTION
**Pull request by ioguix**
Make possible to remove the -w argument using the --old-psql argument.

Fix #270

### CODE REVIEW

**Comment by ioguix** [comment]
ping @rjuju and @frost242

File: check_pgactivity [hunk 1]
```diff
@@ -1062,14 +1081,16 @@ sub query($$;$$$) {
     dprint ("Env. user   : $ENV{PGUSER}    \n") if defined $host->{'user'};
     dprint ("Env. db     : $ENV{PGDATABASE}\n") if defined $host->{'db'};
 
+    $warg = '' if $args{'old-psql'} or $ENV{'CHECK_PGA_OLD_PSQL'};
```
Comment (Krysztophe) [inline check_pgactivity:1084]: CHECK_PGA_OLD_PSQL  is undocumented.

File: check_pgactivity [hunk 2]
```diff
@@ -379,7 +383,20 @@ script relies on the system default temporary directory if possible.
 =item B<-P>, B<--psql> FILE
 
 Path to the C<psql> executable (default: "psql").
-It should be version 8.3 at least, but the server can be older.
+
+Because check_pgactivity uses C<psql -w>, C<psql> must be version 8.4 at least,
+but the server can be older.
+
```
Comment (Krysztophe) [inline check_pgactivity:389]: useless line IMHO, this is the same subject

Comment (ioguix) [reply check_pgactivity:389]: done in next commit.

File: check_pgactivity [hunk 3]
```diff
@@ -379,7 +383,20 @@ script relies on the system default temporary directory if possible.
...
+If your C<psql> is older and does not support C<-w>, look at argument
+C<--old-psql>.
+
+=item B<--old-psql>
+
+Stay compatible with psql 8.3 and below.
+
+This option removes the C<-w> argument from the C<psql> argument list. This
+means that if your password-less authentication is not correctly setup, C<psql>
+will wait indefinitely for a password, effectively blocking C<check_pgactivity>.
```
[the remark this answers is not in the document]
Comment (ioguix) [reply check_pgactivity:399]: fixed in next commit.

File: check_pgactivity (hunk 1 above)
Comment (ioguix) [reply check_pgactivity:1084]: Done in next commit.

**Pull request merged**